lonestar/analytics/cpu/betweennesscentrality/BetweennessCentrality.cpp

```diff
@@ -100,6 +100,11 @@ int main(int argc, char** argv) {
   galois::StatTimer totalTime("TimerTotal");
   totalTime.start();
 
+  galois::graphs::FileGraph degreeGraph;
+  degreeGraph.fromFile(inputFile);
```

Comment:
Can you pass this loaded graph to the 3 variants so that they can load the edges faster?

Reply:
for 2 of the 3, no, because those use the readGraph interface (which I am not familiar with)

for 1 of the 3, yes, but that would require changing FileGraph to detect if the prefix sum (assuming that's what I change the node_degrees above to, which is debatable given what I'm about to write here) is loaded: if it is, it uses the prefix sum array instead of accessing the mmap'd prefix sum on disk. this would require quite a bit of code change

Reply:
Ok, makes sense. This is fine for now.